tasks/piece/task_park_piece.go

```diff
@@ -140,7 +149,8 @@ func (p *ParkPieceTask) Do(taskID harmonytask.TaskID, stillOwned func() bool) (d
 	err = p.db.Select(ctx, &refData, `
         SELECT data_url, data_headers
         FROM parked_piece_refs
-        WHERE piece_id = $1 AND data_url IS NOT NULL`, pieceData.PieceID)
+        WHERE piece_id = $1 AND data_url IS NOT NULL 
+        ORDER BY CASE WHEN data_url LIKE 'file:///%' THEN 0 ELSE 1 END`, pieceData.PieceID)
```

Comment:
If import command has to put the data then it will take 2-5 minutes per command. This is a little impractical. We can remove import command complete. That might a better solution. Running a HTTP server is very easy with Linux. Maybe user's can simply use that.

Reply:
Yes, especially if we ship a tool which is that server in a form that is really easy to run